list.js

```diff
@@ -1,6 +1,57 @@
 (function() {
     var events = [];
     var ITEMS_ON_PAGE = 6;
+    var querySet = [];
+
+    var index = lunr(function() {
+        this.ref('id');
+        this.field('title', {boost: 10});
+        this.field('description', {boost: 5});
+    });
+
+    var doSearch = function() {
+        var q = $('#q').val();
+        var results = index.search(q);
+        var $el = $('#calendarList');
+        $el.empty();
+        $el.show();
+        $el.append('<div class="arrow"></div>');
+        $el.append(
+            $('<h2>RESULTS FOR: "' + q + '"</h2>')
+        );
+        if (results.length === 0) {
+            $el.append('<div class="q-no-item">Unfortunately, there are ' +
+                'no results matching what you\'re looking for in ' +
+                'the Columbia Film Glossary content.</div>');
+        } else {
+            var searchResults = [];
+            for (var r in results) {
+                var e = events[results[r].ref];
```

Comment:
If we don't have to rely on the global scope, it's almost always better not to. I'm going to change `events` here to a local variable so the function is more independent of global state, making it more understandable and more testable.